modules/statistics/php/NDB_Menu_statistics_site.class.inc

```diff
@@ -23,7 +23,16 @@
  */
 class NDB_Menu_Statistics_Site extends NDB_Menu
 {
-
+    /**
+     * Checking user's permission
+     *
+     * @return bool
+     */
+    function _hasAccess()
+    {
+        $user =& User::singleton();
+        return $user->hasPermission('data_entry');
+    }
```

Comment:
Can you move this below the field definitions? It looks a little weird to have a function defined up here before them. And should this have a CenterID check in addition to the data_entry check, since it's a site breakdown?

(Check with @LeighMac to see what she says.)